oauth2.py

```diff
@@ -379,16 +390,31 @@ def create_tweet_payload(text, in_reply_to_tweet_id=None):
             level="info",
         )
         logger.info(publish_alert)
+        store_publication(
+            country_code="Unknown",
+            platform_name=self.platform,
+            source="Platforms",
+            gateway_client="Unknown",
+            status=status,
```

Comment:
We should pass the `published` value directly here instead of using the `status` variable.